src/bp/sdk/botpress.d.ts

```diff
@@ -861,6 +861,11 @@ declare module 'botpress/sdk' {
      * @default 5s
      */
     timeout: string
+    /**
+     * The text limitation of the converse API requests
+     * @default 360
+     */
+    maxMessageLength: number
```

Comment:
For the bot config, we need to specify it in two different locations for the schema building, could you also add this to the bot.config.ts file?